test/e2e/go.mod

```diff
@@ -3,32 +3,32 @@ module github.com/zalando-incubator/kubernetes-on-aws/tests/e2e
 go 1.16
 
 require (
-	github.com/NYTimes/gziphandler v1.0.1 // indirect
 	github.com/evanphx/json-patch v4.9.0+incompatible
 	github.com/gopherjs/gopherjs v0.0.0-20181103185306-d547d1d9531e // indirect
 	github.com/gregjones/httpcache v0.0.0-20181110185634-c63ab54fda8f // indirect
 	github.com/jteeuwen/go-bindata v0.0.0-20151023091102-a0ff2567cfb7
 	github.com/mohae/deepcopy v0.0.0-20170929034955-c48cc78d4826 // indirect
-	github.com/onsi/ginkgo v1.14.1
+	github.com/onsi/ginkgo v1.11.0
 	github.com/onsi/gomega v1.10.2
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/viper v1.7.0
 	github.com/szuecs/routegroup-client v0.17.8-0.20210112151959-1b69df565b42
 	github.com/zalando-incubator/kube-aws-iam-controller v0.1.2
 	gopkg.in/gcfg.v1 v1.2.3 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
-	k8s.io/api v0.20.10
-	k8s.io/apimachinery v0.20.10
+	k8s.io/api v0.18.3
+	k8s.io/apimachinery v0.18.9
 	k8s.io/apiserver v0.0.0
-	k8s.io/client-go v0.20.10
-	k8s.io/kubernetes v0.0.0
+	k8s.io/client-go v0.18.9
+	k8s.io/kubernetes v0.0.0-00010101000000-000000000000
 )
 
 replace (
 	github.com/go-openapi/spec => github.com/go-openapi/spec v0.19.3
 	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.4.1
 	github.com/onsi/ginkgo => github.com/onsi/ginkgo v1.11.0
 	github.com/onsi/gomega => github.com/onsi/gomega v1.7.0
+	github.com/szuecs/routegroup-client => github.com/jonathanbeber/routegroup-client v0.18.9
```

Comment:
temporary until https://github.com/szuecs/routegroup-client/pull/9 is merged and new tag created